lib/ui/window.dart

```diff
@@ -696,6 +800,36 @@ class Window {
     _onDrawFrameZone = Zone.current;
   }
 
+  /// A callback that is invoked to report the [FrameTiming] of recently
+  /// rasterized frames.
+  ///
+  /// This can be used to see if the application has missed frames (through
+  /// [FrameTiming.buildDuration] and [FrameTiming.rasterDuration]), or high
+  /// latencies (through [FrameTiming.totalSpan]).
+  ///
+  /// Unlike [Timeline], the timing information here is available in the release
+  /// mode (additional to the profile and the debug mode). Hence this can be
+  /// used to monitor the application's performance in the wild.
+  ///
+  /// {@macro dart.ui.TimingsCallback.list}
+  ///
+  /// If this is null, no additoinal work will be done. If this is not null,
```

Comment:
nit: typo ("additional")

Reply:
Done. Thanks! Maybe I need a better keyboard... The default logitech one seems to be very easy to swap characters...